### PR TITLE
[ALLUXIO-2295] Fix javadoc in StorageDirView.jav

### PR DESCRIPTION
**Pull request by WSTC2007**
https://alluxio.atlassian.net/browse/ALLUXIO-2295
Update "Id" to "ID" in the javadoc

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

File: core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java [hunk 1]
```diff
@@ -24,7 +24,7 @@
 import javax.annotation.concurrent.NotThreadSafe;
 
 /**
- * This class is a wrapper of {@link StorageDir} to provide more limited access and a filtered list
+ * This class is a wrapper of {@link StorageDir} to provIDe more limited access and a filtered list
```
Comment (gpang) [inline core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java]: This does not have to be modified.

File: core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java [hunk 2]
```diff
@@ -77,8 +77,8 @@ public int getDirViewIndex() {
     List<BlockMeta> filteredList = new ArrayList<>();
 
     for (BlockMeta blockMeta : mDir.getBlocks()) {
-      long blockId = blockMeta.getBlockId();
-      if (mManagerView.isBlockEvictable(blockId)) {
+      long blockID = blockMeta.getBlockID();
```
Comment (gpang) [inline core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java]: you don't have to modify the variable name or method name.

File: core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java [hunk 3]
```diff
@@ -120,8 +120,8 @@ public long getCommittedBytes() {
   public long getEvitableBytes() {
     long bytes = 0;
     for (BlockMeta blockMeta : mDir.getBlocks()) {
-      long blockId = blockMeta.getBlockId();
-      if (mManagerView.isBlockEvictable(blockId)) {
+      long blockID = blockMeta.getBlockID();
```
Comment (gpang) [inline core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java]: you don't have to modify the variable name or method name.

File: core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java [hunk 4]
```diff
@@ -131,22 +131,22 @@ public long getEvitableBytes() {
   /**
    * Clears all marks about blocks to move in/out in this view.
    */
-  public void clearBlockMarks() {
+  public voID clearBlockMarks() {
```
Comment (gpang) [inline core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java]: This does not have to be changed

File: core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java [hunk 5]
```diff
@@ -159,33 +159,33 @@ public StorageTierView getParentTierView() {
...
+    if (mBlocksToMoveIn.add(blockID)) {
       mBlocksToMoveInSize += blockSize;
     }
   }
 
   /**
    * Marks a block to move out of this dir view, which is used by the evictor.
    *
-   * @param blockId the Id of the block
+   * @param blockID the ID of the block
```
Comment (gpang) [inline core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java]: Please only make the `ID` change, not the `blockId` change.

File: core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java [hunk 6]
```diff
@@ -159,33 +159,33 @@ public StorageTierView getParentTierView() {
...
-    return mBlocksToMoveOut.contains(blockId);
+  public boolean isMarkedToMoveOut(long blockID) {
+    return mBlocksToMoveOut.contains(blockID);
   }
 
   /**
    * Marks a block to move into this dir view, which is used by the evictor.
    *
-   * @param blockId the Id of the block
+   * @param blockID the ID of the block
```
Comment (gpang) [inline core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java]: Please only make the `ID` change, not the `blockId` change.

File: core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java [hunk 7]
```diff
@@ -159,33 +159,33 @@ public StorageTierView getParentTierView() {
   /**
    * Returns an indication whether the given block is marked to be moved out.
    *
-   * @param blockId the block id
+   * @param blockID the block ID
```
Comment (gpang) [inline core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java]: Please only make the `ID` change, not the `blockId` change.

File: core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java [hunk 8]
```diff
@@ -159,33 +159,33 @@ public StorageTierView getParentTierView() {
...
   /**
    * Marks a block to move out of this dir view, which is used by the evictor.
    *
-   * @param blockId the Id of the block
+   * @param blockID the ID of the block
    * @param blockSize the block size
    */
-  public void markBlockMoveOut(long blockId, long blockSize) {
-    if (mBlocksToMoveOut.add(blockId)) {
+  public voID markBlockMoveOut(long blockID, long blockSize) {
```
Comment (gpang) [inline core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java]: This does not have to be modified.

File: core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java [hunk 9]
```diff
@@ -159,33 +159,33 @@ public StorageTierView getParentTierView() {
...
   /**
    * Marks a block to move into this dir view, which is used by the evictor.
    *
-   * @param blockId the Id of the block
+   * @param blockID the ID of the block
    * @param blockSize the block size
    */
-  public void markBlockMoveIn(long blockId, long blockSize) {
-    if (mBlocksToMoveIn.add(blockId)) {
+  public voID markBlockMoveIn(long blockID, long blockSize) {
```
Comment (gpang) [inline core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java]: This does not have to be modified.

File: core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java [hunk 10]
```diff
@@ -159,33 +159,33 @@ public StorageTierView getParentTierView() {
   /**
    * Returns an indication whether the given block is marked to be moved out.
    *
-   * @param blockId the block id
+   * @param blockID the block ID
    * @return whether the block is marked to be moved out
    */
-  public boolean isMarkedToMoveOut(long blockId) {
-    return mBlocksToMoveOut.contains(blockId);
+  public boolean isMarkedToMoveOut(long blockID) {
```
Comment (gpang) [inline core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java]: This does not have to be modified.

File: core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java [hunk 11]
```diff
@@ -131,22 +131,22 @@ public long getEvitableBytes() {
...
-  public void clearBlockMarks() {
+  public voID clearBlockMarks() {
     mBlocksToMoveIn.clear();
     mBlocksToMoveOut.clear();
     mBlocksToMoveInSize = mBlocksToMoveOutSize = 0L;
   }
 
   /**
-   * Creates a {@link TempBlockMeta} given sessionId, blockId, and initialBlockSize.
+   * Creates a {@link TempBlockMeta} given sessionID, blockID, and initialBlockSize.
```
Comment (gpang) [inline core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java]: This does not have to be modified.

File: core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java [hunk 12]
```diff
@@ -131,22 +131,22 @@ public long getEvitableBytes() {
...
-   * @param sessionId of the owning session
-   * @param blockId of the new block
+   * @param sessionID of the owning session
+   * @param blockID of the new block
    * @param initialBlockSize of the new block
    * @return a new {@link TempBlockMeta} under the underlying directory
    */
-  public TempBlockMeta createTempBlockMeta(long sessionId, long blockId, long initialBlockSize) {
-    return new TempBlockMeta(sessionId, blockId, initialBlockSize, mDir);
+  public TempBlockMeta createTempBlockMeta(long sessionID, long blockID, long initialBlockSize) {
```
Comment (gpang) [inline core/server/src/main/java/alluxio/worker/block/meta/StorageDirView.java]: This does not have to be modified.

**Comment by ifcharming** [comment]
LGTM, thank you for contributing to Alluxio!

**Pull request merged**